### ISSUE TITLE
Something wrong with bounding box calculation and parallel input

### ISSUE THREAD

**Issue by ericfischer** [mapbox/tippecanoe#251]
There is something flaky about calculating the bounding boxes of features with parallel input.

At bba1c13b0782f, which attempts to detect too-big unclipped features, single-threaded input consistently finds the same features to complain about, as it should:

```
zcat ../buildings.gz | ./tippecanoe -pc -z16 -f -o foo.mbtiles
Warning: foo.mbtiles:89: Large unclipped (-pc) feature will be duplicated across 8 tiles
Warning: foo.mbtiles:3406: Large unclipped (-pc) feature will be duplicated across 30 tiles
Warning: foo.mbtiles:12249: Large unclipped (-pc) feature will be duplicated across 400 tiles
Warning: foo.mbtiles:13404: Large unclipped (-pc) feature will be duplicated across 529 tiles
Warning: foo.mbtiles:15557: Large unclipped (-pc) feature will be duplicated across 1665 tiles
Warning: foo.mbtiles:15580: Large unclipped (-pc) feature will be duplicated across 3888 tiles
Warning: foo.mbtiles:17006: Large unclipped (-pc) feature will be duplicated across 130442 tiles
Exiting because this can't be right.
```

With `-P`, it's expected that a different one might first be seen to cause the problem, but sometimes it makes it all the way through the input without finding any problems at all.

**Comment by ericfischer** [comment]
Looks like this was an uninitialized variable for `maxzoom` when starting the parser thread.

**Comment by ericfischer** [comment]
Fixed in 0f02e9f
